

class LoginError(Exception):
    pass


class ConnectionError(Exception):
    pass
